src/views/Send/SendNFT.vue

```diff
@@ -0,0 +1,631 @@
+    },
+    feesAvailable() {
+      return this.assetFees && Object.keys(this.assetFees).length
+    },
+    isEIP1559Fees() {
+      return cryptoassets[this.asset].chain === this.account?.chain
+    },
+    memoData() {
+      return {
+        memo: this.memo
+      }
+    },
+    asset() {
+      return chains[this.account?.chain].nativeAsset
+    },
+    startAddress() {
+      return this.address.slice(0, 6)
+    },
+    middleAddressPart() {
+      return this.address.substring(6, this.address.length - 4)
+    },
+    endAddress() {
+      return this.address.slice(this.address.length - 4)
+    }
+  },
+  methods: {
+    ...mapActions(['sendNFTTransaction', 'updateFees', 'trackAnalytics']),
+    getAssetIcon,
+    shortenAddress,
+    formatFiat,
+    formatFiatUI,
+    prettyBalance,
+    getFeeAsset,
+    getSendFee,
+    getFeeLabel,
+    getNativeAsset,
+    applyFilters(filters) {
+      this.activityData = applyActivityFilters([...this.assetHistory], filters)
+    },
+    async copy(text) {
+      await navigator.clipboard.writeText(text)
+    },
+    selectNFT(asset) {
+      this.selectedNFT = asset
+    },
+    closeSendErrorModal() {
+      this.sendErrorModalOpen = false
+      this.loading = false
+    },
+    next(view) {
+      this.activeView = view
+    },
+    setActiveView(view) {
+      if (this.activeView === 'selectAsset') {
+        return '/wallet/nfts'
+      }
+      this.activeView = view
+    },
+    cancelCustomFee() {
+      this.activeView = 'selectedAsset'
+      this.selectedFee = 'average'
+    },
+    setCustomFee: _.debounce(async function ({ fee }) {
+      this.customFee = fee
+      this.updateSendFees(this.amount)
+    }, 800),
+    applyCustomFee({ fee }) {
+      const presetFee = Object.entries(this.assetFees).find(
+        ([speed, speedFee]) =>
+          speed !== 'custom' &&
+          (speedFee.fee === fee ||
+            (fee.maxPriorityFeePerGas &&
+              speedFee.fee.maxPriorityFeePerGas === fee.maxPriorityFeePerGas &&
+              speedFee.fee.maxFeePerGas === fee.maxFeePerGas))
+      )
+
+      if (presetFee) {
+        const [speed] = presetFee
+        this.selectedFee = speed
+        this.customFee = null
+      } else {
+        this.updateMaxSendFees()
+        this.updateSendFees(this.amount)
+        this.customFee = typeof fee === 'object' ? fee.maxFeePerGas + fee.maxPriorityFeePerGas : fee
+        this.selectedFee = 'custom'
+      }
+      this.activeView = 'selectedAsset'
+    },
+    onCustomFeeSelected() {
+      this.activeView = 'custom-fees'
+    },
+    resetCustomFee() {
+      this.customFee = null
+      this.selectedFee = 'average'
+    },
+    async _updateSendFees(amount) {
+      const getMax = amount === undefined
+      if (this.feesAvailable) {
+        const sendFees = {}
+
+        for (const [speed, fee] of Object.entries(this.assetFees)) {
+          const feePrice = fee.fee.maxPriorityFeePerGas + fee.fee.suggestedBaseFeePerGas || fee.fee
```

Comment:
You are reusing an old `feePrice` formula from other screen. There is a new one.